cmd/gofuse/main.go

```diff
@@ -152,6 +152,22 @@ func run() error {
 		return filepath.Join(fusedRoot, "src", trimUpTo(rel(projectRoot, path), "github.com"))
 	})
 
+	templateGenedGofiles := collect(binOut, always).ifTrue(and(
```

Comment:
Add comment to say which files are retrieved here, e.g.:
`// Get ".go" files generated from templates`

And similar a few lines below:
`// Get ".cpp" and ".h" files generated from templates`